actionview/lib/action_view/context.rb

```diff
@@ -17,7 +17,7 @@ module Context
     # Prepares the context by setting the appropriate instance variables.
     def _prepare_context
       @view_flow     = OutputFlow.new
-      @output_buffer = nil
+      @output_buffer = ActionView::OutputBuffer.new
```

Comment:
This isn't a test only change, but I think it's for the best. The goal ultimately is to stop swapping the buffer, so ultimately allocating it when the context is allocated will make sense.